examples/step-54/step-54.cc

```diff
@@ -170,6 +170,7 @@ namespace Step54
   void TriangulationOnCAD::read_domain()
   {
     TopoDS_Shape bow_surface = OpenCASCADE::read_IGES(cad_file_name, 1e-3);
+    std::cout << " Read " << cad_file_name << std::endl;
```

Comment:
```suggestion
    std::cout << "  Read " << cad_file_name << std::endl;
```
I would go with 2 spaces.